aeat.py

```diff
@@ -507,6 +508,9 @@ def load_invoices(cls, reports):
 
         to_create = []
         for report in reports:
+            if not report.load_date:
+                report.load_date = date.today()
+                report.save()
```

Comment:
If this condition is added, it's not necessary the conditional in the line 540. The domain could be added directly, became the report.load_date exist.